blockindex/contractstaking/indexer.go

```diff
@@ -52,6 +54,7 @@ type (
 		cache                *contractStakingCache // in-memory index for clean data, used to query index data
 		contractAddress      string                // stake contract address
 		contractDeployHeight uint64                // height of the contract deployment
+		height               atomic.Value          // uint64, current block height
```

Comment:
use `atomic.Uint64`

Reply:
`atomic.Uint64` is a new atomic types introduced in [go 1.19](https://tip.golang.org/doc/go1.19#atomic_types), while our CI is using an older version, 1.18.5.